site-scrapers/CVS.js

```diff
@@ -28,21 +28,39 @@ module.exports = async function GetAvailableAppointments(browser) {
         `${webData.responsePayloadData.currentTime}-0${offsetMountain}:00`
     );
     return webData.responsePayloadData.data.MA.map((responseLocation) => {
-        let hasAvailability = parseInt(responseLocation.totalAvailable)
-            ? true
-            : false;
-        let totalAvailability = parseInt(responseLocation.totalAvailable);
-        let availability = {};
+        // Prior to Feb 22 or so, CVS's JSON returned:
+        //  {
+        //    "totalAvailable": "0",
+        //    "city": "CAMBRIDGE",
+        //    "state": "MA",
+        //    "pctAvailable": "0.00%",
+        //    "status": "Fully Booked"
+        //  },
+        //
+        // But as of Feb. 25 it returns:
+        //  {
+        //    "city": "CAMBRIDGE",
+        //    "state": "MA",
+        //    "status": "Fully Booked"
+        //  },
+        // It's not clear whether we can expect the other fields to return when there is
+        // availability.
+        // Also, we had previously seen cases where numeric availability was "0" but status
+        // was "Available" and it's not apparent what that really meant (skew? bugs?).
         responseLocation.city = toTitleCase(responseLocation.city);
-        return {
-            name: `${siteName} (${responseLocation.city})`,
-            hasAvailability,
-            availability,
-            totalAvailability,
-            timestamp: timestamp,
-            signUpLink: site.website,
-            ...responseLocation,
-        };
+        responseLocation.name = `${siteName} (${responseLocation.city})`;
+        responseLocation.hasAvailability =
+            responseLocation.status !== "Fully Booked";
+        let totalAvailability =
+            responseLocation.totalAvailable &&
+            parseInt(responseLocation.totalAvailable);
+        if (totalAvailability) {
+            responseLocation.totalAvailability = totalAvailability;
+        }
+        responseLocation.availability = {};
+        responseLocation.timestamp = timestamp;
+        responseLocation.signUpLink = site.website;
```

Comment:
A couple things:
1) I don't think we need to set all the object properties in different statements. Instead, we could do
```
return {
  name: `${siteName} blah`,
  hasAvailability: responseLocation.status !== "Fully Booked",
  totalAvailability: responseLocation.totalAvailable ? parseInt(responseLocation.totalAvailable) : 0,
  ...etc...
  ...responseLocation,
};
```
2) I don't see the need to update hasAvailability to check the status instead of totalAvailability unless we've seen bugs already due to that logic checking totalAvailability.
3) Do you mind updating `...responseLocation` to explicitly return only the properties we care about? I don't love that we're throwing in random fields like "PctAvailable" and stuff. We can punt on that though 🤷

Reply:
1: That's fine if you're willing to return 0 for `totalAvailability`. The whole point of `039abd` was to avoid returning it at all if we didn't know a number.
2. Given that `totalAvailable` is no longer being returned now (with `status` "Fully Booked"), I don't think we can assume it will return when `status` changes to "Available" (or whatever)
3. HA! I thought it was a **feature** to return CVS's JSON with random fields we don't need. In particular, it means that our saved `data.json` files are available for forensic trawling without having to institute a separate snapshotting mechanism. But if you disagree, happy to go with that.

Reply:
I'd prefer to have debugging efforts logged separately - we've had some bugs due to passing wrong fields, etc., so I want to attempt to enforce a structure here. I have a unit test in progress that will address this in a separate branch.

Reply:
Done as to (3) (not returning extraneous data): 9d81ac0dca21e205bef19252064e9ba7430cfc2b
Still waiting for your judgment on (1) and (2).

Reply:
I'm fine with leaving those things